packages/alpha/CHANGELOG.md

```diff
@@ -3,82 +3,121 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-# [1.0.0-rc.10](https://github.com/cognitedata/cognite-sdk-js/compare/@cognite/sdk-alpha@1.0.0-rc.9...@cognite/sdk-alpha@1.0.0-rc.10) (2025-01-31)
+## [0.35.2](https://github.com/cognitedata/cognite-sdk-js/compare/@cognite/sdk-alpha@0.35.1...@cognite/sdk-alpha@0.35.2) (2025-01-17)
 
-**Note:** Version bump only for package @cognite/sdk-alpha
 
+### Bug Fixes
 
+* **simint:** add simulation run example ([#1188](https://github.com/cognitedata/cognite-sdk-js/issues/1188)) ([a912037](https://github.com/cognitedata/cognite-sdk-js/commit/a912037c1c4afaddffef6d644decc4ec561b6def))
```

Comment:
think we should add this change log under 9x ? or do you think to do it afer merge ?